Libraries/Network/RCTNetworking.m

```diff
@@ -313,26 +315,16 @@ - (RCTURLRequestCancellationBlock)processDataForHTTPQuery:(nullable NSDictionary
   return callback(nil, nil);
 }
 
-- (void)sendData:(NSData *)data forTask:(RCTNetworkTask *)task
+- (NSString *)decodeTextData:(NSData *)data fromResponse:(NSURLResponse *)response
```

Comment:
Since this doesn't depend on any state now, could you just it make it a static function?